frontend/src/scenes/instance/SystemStatus/OverviewTab.tsx

```diff
@@ -34,6 +34,10 @@ function RenderValue(metricRow: MetricRow): JSX.Element | string {
         return <Tag color={value ? 'success' : 'error'}>{value ? 'Yes' : 'No'}</Tag>
     }
 
+    if (typeof value === 'number') {
+        return formatNumberWithSeparator(value)
```

Comment:
stackOverflow suggests an easier way https://stackoverflow.com/questions/1823058/how-to-print-number-with-commas-as-thousands-separators
```suggestion
        return f'{value:,}'
```

Reply:
That's Python though?

But yeah sure I can search for the "standard" JS way - I just wrote this myself.

Reply:
Does seem like `toLocaleString("en-US")` does the trick. I was working without an internet connection so just wrote it my way lol

Reply:
oh 🤦‍♀️  not sure why I thought it was python ... other files I guess